### PR TITLE
Dist fork: use ovs-kubernetes fork (not file patches)

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

**Comment by hunchback** [comment]
running now:

```
vagrant@vagrant:~/projects/openvswitch/ovsdb-etcd$ make ovn-get
kubectl --kubeconfig=/home/vagrant/admin.conf -n=ovn-kubernetes get pods
NAME                              READY   STATUS    RESTARTS   AGE
ovnkube-db-656d5bb4b4-rmptg       3/3     Running   0          9m59s
ovnkube-master-776579fc66-qbxdt   3/3     Running   2          9m59s
ovnkube-node-5xt7b                2/3     Running   1          9m58s
ovnkube-node-866tr                2/3     Running   1          9m58s
ovnkube-node-9zc7c                2/3     Running   2          9m58s
ovs-node-thhsg                    1/1     Running   0          9m59s
ovs-node-vbfzh                    0/1     Running   0          9m59s
ovs-node-zhlkj                    1/1     Running   0          9m59s
```

File: Makefile [hunk 1]
```diff
@@ -69,42 +69,41 @@ south-server:
...
+	cp ${OVN_KUBERNETES_ROOT}/dist/images/ovndb-raft-functions.sh dist/images/.
+	cp ${OVN_KUBERNETES_ROOT}/dist/images/ovnkube.sh dist/images/.
 	docker build . -t etcd -f dist/images/Dockerfile.etcd
+	docker build . -t ovsdb-etcd -f dist/images/Dockerfile.ovsdb-etcd
 
-.PHONY: image-server
-image-server: build
-	docker build . -t server -f dist/images/Dockerfile.ovsdb-etcd
-
-.PHONY: ovnk
```
Comment (roytman) [inline Makefile:80]: can you return the ovnk rule, but without ovnk-deploy ?

File: dist/README.md [hunk 2]
```diff
@@ -1,79 +1,79 @@
...
-git clone https://github.com/hunchback/ovn
-cd ovn
-git checkout ovndb-etcd
-./boot.sh
-./configure
-make
-export OVN_ROOT=$PWD
-```
 
 ## Prepare ovs-kubernetes
```
Comment (roytman) [inline dist/README.md:4]: Prepare ovn-kubernetes

File: dist/README.md [hunk 3]
```diff
@@ -1,79 +1,79 @@
...
-```bash
-cp $OVN_ROOT/utilities/ovn-nbctl dist/images/
-make -C dist/images fedora
-```
-
-## Build ovn-kubernetes
+## Build docker images
 
-clone and build and push (aboute 5 minutes):
+clone and build and push:
```
Comment (roytman) [inline dist/README.md:16]: should we mention cd .. (or something similar) ? I mean not to clone ovsdb-etcd into ovn-kubernetes

File: dist/README.md [hunk 4]
```diff
@@ -1,79 +1,79 @@
...
-make ovnk-deploy
+make ovn-status
 ```
 
-After the deplyment is complete you can check out the relevant pods.
+Or directly:
 
 ```bash
-make ovnk-status
+export KUBECONFIG=${HOME}/admin.conf
```
Comment (roytman) [inline dist/README.md:77]: we have done this export several times

File: dist/README.md [hunk 5]
```diff
@@ -1,79 +1,79 @@
...
+kubectl --kubeconfig=${KUBECONFIG} -n=ovn-kubernetes get pods
```
Comment (roytman) [inline dist/README.md:78]: if you defined KUBECONFIG, you don't have to add --kubeconfig=${KUBECONFIG}

**Pull request merged**